src/applications/personalization/rated-disabilities/components/RatedDisabilityView.jsx

```diff
@@ -81,9 +116,29 @@ class RatedDisabilityView extends React.Component {
                 fetchRatedDisabilities={fetchRatedDisabilities}
                 ratedDisabilities={ratedDisabilities}
               />
-            </div>
-            <div className="vads-l-col--12 medium-screen:vads-l-col--4">
-              <RatedDisabilitiesSidebar />
+              <h3
+                id="learn"
+                className="vads-u-padding-bottom--1p5 vads-u-border-bottom--3px vads-u-border-color--primary vads-u-font-size--h2"
+              >
+                Learn about VA disability ratings
+              </h3>
+              <p>
+                To learn how we determined your VA combined disability rating,
+                use our disability rating calculator and ratings table.
+              </p>
+              <a
+                href="/disability/about-disability-ratings/"
+                aria-label="About VA disability ratings"
```

Comment:
No need for the aria-label if it matches the link text

Reply:
@timwright12 Good catch! Removed this